virtual_rainforest/models/abiotic_simple/abiotic_simple_model.py

```diff
@@ -0,0 +1,232 @@
+"""The :mod:`~virtual_rainforest.models.abiotic_simple.abiotic_simple_model` module
+creates a
+:class:`~virtual_rainforest.models.abiotic_simple.abiotic_simple_model.AbioticSimpleModel`
+class as a child of the :class:`~virtual_rainforest.core.base_model.BaseModel` class.
+At present a lot of the abstract methods of the parent class (e.g.
+:func:`~virtual_rainforest.core.base_model.BaseModel.spinup`) are overwritten using
+placeholder functions that don't do anything. This will change as the Virtual Rainforest
+model develops. The factory method
+:func:`~virtual_rainforest.models.abiotic_simple.abiotic_simple_model.AbioticSimpleModel.from_config`
+exists in a
+more complete state, and unpacks a small number of parameters from our currently pretty
+minimal configuration dictionary. These parameters are then used to generate a class
+instance. If errors crop here when converting the information from the config dictionary
+to the required types they are caught and then logged, and at the end of the unpacking
+an error is thrown. This error should be caught and handled by downstream functions so
+that all model configuration failures can be reported as one.
+"""  # noqa: D205, D415
+
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+from pint import Quantity
+from xarray import DataArray
+
+from virtual_rainforest.core.base_model import BaseModel
+from virtual_rainforest.core.data import Data
+from virtual_rainforest.core.exceptions import InitialisationError
+from virtual_rainforest.core.logger import LOGGER
+
+# The simple_regression.py is called by self.setup and self.update, will follow soon
+# from virtual_rainforest.models.abiotic_simple import simple_regression
+
+
+class AbioticSimpleModel(BaseModel):
+    """A class describing the abiotic simple model.
+
+    Args:
+        data: The data object to be used in the model.
+        update_interval: Time to wait between updates of the model state.
+        soil_layers: The number of soil layers to be modelled.
+        canopy_layers: The initial number of canopy layers to be modelled.
+        initial_soil_moisture: The initial soil moisture for all layers.
+    """
+
+    model_name = "abiotic_simple"
+    """The model name for use in registering the model and logging."""
+    lower_bound_on_time_scale = "1 day"
+    """Shortest time scale that soil model can sensibly capture."""
```

Comment:
"that the abiotic simple model" I guess? Also same comment for the doc string below

Reply:
upsi, yes, thanks :-)